docs/tutorial/tut_point_in_time.md

```diff
@@ -51,75 +62,178 @@ Create a new dbt model as before. We'll call this one `pit_customer`.
 To create a PIT model, we simply copy and paste the above template into a model named after the PIT we
 are creating. dbtvault will generate a PIT using parameters provided in the next steps.
 
-PITS should use the pit_incremental materialization, as the pit is remade with every new as of dates table. 
+PIT tables should use the *pit_incremental* materialization, as they will be remade with every new As-of Dates table. 
 
 We recommend setting the `pit_incremental` materialization on all of your pits using the `dbt_project.yml` file:
 
-`dbt_project.yml`
+=== "dbt_project.yml"
+
+    ```yaml
+    models:
+      my_dbtvault_project:
+       pit:
+        materialized: pit_incremental
+        tags:
+          - pit
+        pit_customer:
+          vars:
+            ...
+    ```
+
+### Adding the metadata
+
+Let's look at the metadata we need to provide to the [pit](../metadata.md#point-in-time-pit-tables) macro.
+
+#### Source table
+
+Here we will define the metadata for the source_model. We will use the HUB_CUSTOMER that we built before.
+
+=== "dbt_project.yml"
 
-```yaml
-models:
-  my_dbtvault_project:
-   pit:
-    materialized: pit_incremental
-    tags:
-      - pit
+    ```yaml
     pit_customer:
       vars:
+        source_model: HUB_CUSTOMER
         ...
-```
+    ```
 
-### Adding the metadata
+#### Primary Key
 
-Let's look at the metadata we need to provide to the [pit](../macros.md#pit) macro.
+Next we need add the Hub's Primary Key column 
 
-#### Source table
-Here we will define the metadata for the source_model. We will use the HUB_CUSTOMER that we built before.
+=== "dbt_project.yml"
+
+    ```yaml
+    pit_customer:
+      vars:
+        source_model: HUB_CUSTOMER
+        src_pk: CUSTOMER_PK
+        ...
+    ```
+
+#### As-of Date Table 
 
-`dbt_project.yml`
+Next, we provide the PIT's column name for the as-of dates.
```

Comment:
Is this the column name OR the name of the table we want to get the dates from?

Reply:
date_spine makes the column name containing the date, `DATE` are PITs hard coded to use `AS_OF_DATE(S)` or is it configurable? Do we just provide the name of the table and the PIT assumes it contains an `AS_OF_DATE` column?

Reply:
That phrase is completely erroneous. It should have said "Next, we provide the name of the As of Dates table". That's what the `as_of_dates_table` parameter expects: the name of the As of Dates table. 

Regarding `AS_OF_DATE`, that is the default (ie. hard coded) name of the date column in the As of Dates table that the PIT macro expects. 

`pit macro`

`
[..]
as_of_grain_old_entries AS (
    SELECT DISTINCT **AS_OF_DATE** FROM {{ this }}
),
[...]
`

So, when you generate the As of Dates table it has to have its date/datetime column named to `AS_OF_DATE`. Otherwise, it will throw an error when running the PIT model.

Reply:
Resolved